app/src/main/java/org/desarrolladorslp/teckersapp/ui/messages/MessageAdapter.kt

```diff
@@ -72,7 +72,7 @@ data class MessageAdapter(private val messagesHeader: ArrayList<MessageHeader>)
                 Picasso.get()
                     .load(messageheader.senderImage)
                     .centerCrop()
-                    .transform(CircleTransform(30, 0))
+                    .transform(CircleTransform(120, 0))
```

Comment:
Why 120? If the view is a 60dp x 60dp it would make sense that the radius is 30.

Reply:
okey i put it back